expr/builtins/hash_and_encode.go

```diff
@@ -186,3 +186,22 @@ func encodeB64DecodeEval(ctx expr.EvalContext, args []value.Value) (value.Value,
 	}
 	return value.NewStringValue(string(by)), true
 }
+
+// RandomSalt generates a random ten-digit number as a string
+//
```

Comment:
You need to register your function as well here: https://github.com/lytics/qlbridge/blob/85f454149b85d2165e7a003035d5c95ef051fc66/expr/builtins/builtins.go#L127

something like:
```
		expr.FuncAdd("random.salt", &RandomSalt{})
```